doc: explicitly mention node:fs module restriction (#54269)
nodejs/node: doc/api/permissions.md

#### File System Permissions

> The Permission Model by default restricts access to `node:fs` module.
> It doesn't guarantee users won't be able to write to the disk.
> For instance, `node:sqlite` allows users to use
> a file-based database.

Reviewer: ```suggestion
The Permission Model, by default, restricts access to the file system through the `node:fs` module.
It does not guarantee that users will not be able to access the file system through other means, such as through the `node:sqlite` module.
```
